control/control.SLES.xml

```diff
@@ -273,6 +307,25 @@ Please visit us at http://www.suse.com/.
 
         <!-- FATE #305583: Start CIMOM by default -->
         <service_sfcb><label>CIM Server</label></service_sfcb>
+
+        <normal_role>
+          <label>Default System</label>
+        </normal_role>
+        <normal_role_description>
+          <label>Nothing is modified compared to plain installation</label>
+        </normal_role_description>
+        <kvm_host_role>
+          <label>KVM Virtualization Host</label>
+        </kvm_host_role>
+        <kvm_host_role_description>
+          <label>A system with KVM pattern, no /home</label>
+        </kvm_host_role_description>
+        <xen_host_role>
+          <label>Xen Virtualization Host</label>
+        </xen_host_role>
+        <xen_host_role_description>
+          <label>A system with Xen pattern, no /home</label>
```

Comment:
do we really plan to use these texts? I found `no /home` very confusing. Better is something like `Does not propose separated /home partition.`
Also I think it should be more benefits oriented like
```
A system with pre-installed xen pattern for smoother experience.  Propose to not use separated /home partition as virtual hosts does not benefit from it.
```